airflow_valohai_plugin/operators/valohai_download_execution_outputs_operator.py

```diff
@@ -0,0 +1,56 @@
+import os
+import re
+from urllib.request import urlretrieve
+import logging
+
+from airflow.models import BaseOperator
+from airflow.configuration import AIRFLOW_HOME
+
+
+class ValohaiDownloadExecutionOutputsOperator(BaseOperator):
+    """
+    Downloads outputs locally from a ValohaiSubmitExecutionOperator
+    tasks that stored outputs details in XCOM.
+
+    Args:
+        output_task_id (str): id of the ValohaiSubmitExecutionOperator
+            task that produced the outputs.
+        output_path (str): relative path to AIRFLOW_HOME where to
+            store the outputs locally.
+        output_name_pattern (str, optional): name or regex pattern to filter
+            output names to donwload.
+    """
+    ui_color = '#002f6c'
+    ui_fgcolor = '#fff'
+
+    def __init__(
+        self,
+        output_task_id,
+        output_path,
+        output_name_pattern=None,
+        *args,
+        **kwargs
+    ):
+        super(ValohaiDownloadExecutionOutputsOperator, self).__init__(*args, **kwargs)
+        self.output_task_id = output_task_id
+        self.output_path = output_path
+        self.output_name_pattern = output_name_pattern
+
+    def get_output_path(self, name):
+        return os.path.join(AIRFLOW_HOME, self.output_path, name)
```

Comment:
@jdiprizio wondering whether this is fine for our use case where we have the /data folder?

Reply:
In our use case, I think the data folder is mounted as a subdirectory of `AIRFLOW_HOME` so this should be fine. Wondering if we want to allow outputting outside of `AIRFLOW_HOME`?

Reply:
Thanks. Indeed, outputting inside of AIRFLOW_HOME should be fine.